### PR TITLE
[hlm-621] - Individual

### PR DESCRIPTION
**Pull request by jayantp-egov**

### CODE REVIEW

File: health-services/individual/src/main/java/org/egov/individual/service/IndividualService.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            return uuidList;
+        };
+    }
+
+    private static Individual enrichWithSystemGeneratedIdentifier(Individual individual) {
+        if (individual.getIdentifiers() == null || individual.getIdentifiers().isEmpty()) {
+            List<Identifier> identifiers = new ArrayList<>();
+            identifiers.add(Identifier.builder()
+                    .identifierType("SYSTEM_GENERATED")
+                    .identifierId(UUID.randomUUID().toString())
```
Comment (egov-joy) [inline health-services/individual/src/main/java/org/egov/individual/service/IndividualService.java:93]: Let's use IDGen for system generated registry identifiers

File: health-services/individual/src/main/resources/db/migration/main/V20221227123000__individual_individual_create_ddl.sql [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+(
+    id                character varying(64),
+    userId            character varying(64),
+    clientReferenceId character varying(64),
+    tenantId          character varying(1000),
+    givenName         character varying(200),
+    familyName        character varying(200),
+    otherNames        character varying(200),
+    -- YYYYMMDD
+    dateOfBirth       character varying(8),
```
Comment (egov-joy) [inline health-services/individual/src/main/resources/db/migration/main/V20221227123000__individual_individual_create_ddl.sql:11]: I'd suggest we use `date` datatype so future operations such as search based on age etc are easier.

File: health-services/individual/src/main/java/org/egov/individual/service/IndividualService.java (hunk 1 above)
Comment (jayantp-egov) [reply health-services/individual/src/main/java/org/egov/individual/service/IndividualService.java]: Done.

File: health-services/individual/src/main/resources/db/migration/main/V20221227123000__individual_individual_create_ddl.sql (hunk 2 above)
Comment (jayantp-egov) [reply health-services/individual/src/main/resources/db/migration/main/V20221227123000__individual_individual_create_ddl.sql]: Done

**Review by egov-joy — APPROVED** [review]

**Pull request merged**